assignments/objects.js

```diff
@@ -49,16 +90,39 @@ let example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-let parent = {}
+let parent = {
+  'name': 'Susan',
+  'age': 70,
+  'child': {
+    'name': 'George',
+    'age': 50,
+    'grandchild': {
+      'name': 'Sam',
+      'age': 30,
+      speak: function () {
```

Comment:
I like how you organized this so that the methods were included _after_ the nested objects. It looks nice and it helps to visually break up the cascade of closing brackets.